src/Swarm/Game/Step.hs

```diff
@@ -927,7 +927,10 @@ execConst c vs s k = do
     Count -> case vs of
       [VString name] -> do
         inv <- use robotInventory
-        return $ Out (VInt (fromIntegral $ countByName name inv)) s k
+        let n = case countByName name inv of
+              E.Count x -> x
+              E.Infinity -> 42
```

Comment:
:laughing:

Reply:
If we go the pos/neg infinity way, we could give players the ability to count with actual infinities assuming they have something infinite in inventory (or they divide by zero?).

But without it, I could not really come up with a meaningful answer.

Reply:
Yeah, and it would be annoying for `count` to always return something of type `() + int` just to deal with the possibility of infinity.  But at least it should be a really big number like a million or a billion or something like that, instead of 42.

Reply:
This looks like a good candidate for an integration test. :slightly_smiling_face: 

We can test that the base in the creative scenario has an infinite count of items.

Reply:
:heavy_check_mark: